src/questions/general-questions.md

```diff
@@ -34,3 +34,10 @@ permalink: /questions/general-questions/index.html
 * What role do you see yourself in?
 * Explain the difference between cookies, session storage, and local storage?
 * Can you explain what happens when you enter a URL into the browser?
+* Can you describe the components of a web application?
```

Comment:
Not sure what this answer would actually be. I could literally write  a whole book answering this. Or someone could say "HTML + CSS + JavaScript" and then shrug.

Reply:
yes I agree it's very broad, but I think it's for testing your instinct and excitement overall about web development. you can start with "server and client" as components and go a little more deep into that topic, covering the basics